CHANGELOG.md

```diff
@@ -74,6 +77,14 @@ different versioning scheme, following the Haskell community's
 * Added `bond::blob_prolong` helper function that will return a `bond::blob` with
   a copied data if the original one does not own the memory.
 * The `bond::OutputBuffer::GetBuffers` now can accept arbitrary STL-like containers.
+* `bond::maybe<T>` has been overhauled.
+    * Fixed a bug that default initialized an instance of `T` even when a maybe held nothing.
+    * Added `noexcept` variants of `bond::maybe<T>::value`.
+    * Added `bond::maybe<T>::emplace` to construct a maybe's value in place.
+    * Added various rvalue-reference and allocator-aware constructors and
+      assignment operators.
+    * Added `bond::maybe<T>::operator==(const T&)` to comapre directly to
```

Comment:
The `operator==` is not a member anymore.

Reply:
✔️